ci/prometheus-down.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# 
+# This script checks a few known queries against the internal prometheus query
+# API.  These queries are chosen because they have timestamps as their values.
+# The queries and the prometheus host can be overridden with environment 
+# variables.
+# 
+# If the query fails, then something is wrong with Prometheus's query API.
+# 
+# If at least one timestamp is up to date, then we know it's probably working,
+# otherwise, something is jammed up inside probably.
+# 
+# 
+# XXX If something stops the processes that generate these timestamp metrics,
+# this will result in false positives.  Maybe a deploy might do this?
+#
+
+set -u
+
+: ${PROMETHEUSHOST:="0.prometheus.production-monitoring.prometheus-production.toolingbosh"}
+: ${QUERIES:="
+  concourse_has_opslogin_lastcheck
+  bosh_unknown_iaas_instance_lastcheck
+  bosh_last_scrape_timestamp
+  aws_iam_user_lastcheck
+"}
+
+TIME=$(date +%s)
+APIOK=yes
```

Comment:
I think that this had to do with the way I did the logic originally, which changed as I found more ways it could break.  I don't think it matters either way, though.